lib/config/index.js

```diff
@@ -34,25 +34,20 @@ const configFilePath = path.resolve(appRootPath, process.env.CMD_CONFIG_FILE ||
 'config.json')
 const fileConfig = fs.existsSync(configFilePath) ? require(configFilePath)[env] : undefined
 
-let config = require('./default')
-merge(config, require('./defaultSSL'))
-merge(config, require('./oldDefault'))
-merge(config, debugConfig)
-merge(config, packageConfig)
-merge(config, fileConfig)
-merge(config, require('./oldEnvironment'))
-merge(config, require('./hackmdEnvironment'))
-merge(config, require('./environment'))
-merge(config, require('./dockerSecret'))
-
-if (['debug', 'verbose', 'info', 'warn', 'error'].includes(config.loglevel)) {
+let config = require('./convict')
+config.load(debugConfig)
+config.load(packageConfig)
+config.load(fileConfig)
+config.load(require('./dockerSecret'))
+
+if (['debug', 'verbose', 'info', 'warn', 'error'].includes(config.get('loglevel'))) {
   logger.level = config.loglevel
 } else {
   logger.error('Selected loglevel %s doesn\'t exist, using default level \'debug\'. Available options: debug, verbose, info, warn, error', config.loglevel)
 }
```

Comment:
this sanity check is already in convict config, including the error message. It should be correct at this point.